### DISCUSSION TITLE
Could I ignore the commit activity for fork repository?

### DISCUSSION THREAD

**Discussion by razonyang** [Readme-Workflows/recent-activity#283] [Q&A] [answered]
Hi, is there an option to ignore the commits of fork repository?

![image](https://user-images.githubusercontent.com/17720932/236669398-b89964cf-81d3-496b-809b-59a29fd8dfdb.png)

**Comment by abhijoshi2k** [comment, marked as answer]
There's no option to ignore forked repositories specifically. However, you can blacklist certain repositories from getting listed. So you can add your forked repos to that list in config. Hope this answers your question.

**Reply by razonyang** [reply]
Thanks for your reply, but it may be useful to add an option to ignore any forks commits, `ignored_repos` seems hard to maintain.
